package.nls.json

```diff
@@ -148,7 +148,7 @@
     "cmake-tools.configuration.cmake.allowCommentsInPresetsFile.description": "Allow the use of JSON extensions such as comments in CMakePresets.json. Please note that your CMakePresets.json file may be considered invalid by other IDEs or on the command line if you use non-standard JSON.",
     "cmake-tools.configuration.cmake.ignoreCMakeListsMissing.description": { "message": "If `true`, the extension will not ask the user to select a CMakeLists.txt file for configuration when one is found in the workspace but not in the root folder.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "cmake-tools.configuration.cmake.launchBehavior.description": "Controls what happens with the launch terminal when you launch a target.",
-    "cmake-tools.configuration.cmake.launchBehavior.reuseTerminal.description": "The launch terminal instance is reused and the target will launch as soon as the terminal is idle.",
-    "cmake-tools.configuration.cmake.launchBehavior.breakAndReuseTerminal.description": "The launch terminal instance is reused and a break command is sent to terminate any active foreground process before launching the target.",
-    "cmake-tools.configuration.cmake.launchBehavior.newTerminal.description": "A new terminal instance is created and the target is launched in it. Existing terminals are not automatically cleaned up."
+    "cmake-tools.configuration.cmake.launchBehavior.reuseTerminal.markdownDescriptions": { "message":"The launch terminal instance is reused and the target will launch as soon as the terminal is idle.", "comment": [] },
```

Comment:
If it doesn't need a comment, should these be left alone (line 151 & 153)?

Reply:
They were a part of the same `markdownEnumDescriptions`.

Reply:
do markdown descriptions have to have a comment?

Reply:
it looks like it works without a comment (I tested it).
It didn't work if I define one in `"markdownEnumDescriptions"`  and the other two in `"enumDescriptions"`, 
but it works if I define all 3 in `"markdownEnumDescriptions"` and don't add `comment` to the other two.